common/check_for_and_setup_RACF_authorizations.md

```diff
@@ -0,0 +1,34 @@
+---

```

Comment:
Change file name to `setup_precheck.md`

Reply:
changed and changed corresponding links.